src/entities/redemption-info.ts

```diff
@@ -0,0 +1,23 @@
+import type { CheckoutStatusResponse } from "../networking/responses/checkout-status-response";
+
+/**
+ * This object gives you access to the purchase redemption data.
```

Comment:
Maybe a bit more detail like:
```suggestion
 * This object gives you access to the purchase redemption data when
 * the purchase can be redeemed to a mobile user, like in the case of anonymous users.
```